fn_ipvoid/fn_ipvoid/util/selftest.py

```diff
@@ -0,0 +1,56 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation
+   test with: resilient-circuits selftest -l fn_ipvoid
+"""
+
+import logging
+import requests
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+log.addHandler(logging.StreamHandler())
+
+BASE_URL = 'https://endpoint.apivoid.com/iprep/v1/pay-as-you-go/?key={}&stats'
+
+def get_config_option(option_name, opts, optional=False):
```

Comment:
`get_config_option` could be added to `ipvoid_helper`

You could also use `get_config_option` in `function_ipvoid.py` when getting the API key and Base URL